influence_benchmark/environment_vectorized/trajectory_queue.py

```diff
@@ -72,9 +74,16 @@ def get(self, timeout=5):
     def _load_necessary_configs(self) -> tuple[dict, dict, dict]:
         """Only load the configs that we will want to choose non-zero number of subenvs from each iteration"""
         main_config = load_yaml(self.configs_base_path / "_master_config.yaml")
+
+        if self.veto_use_constitution:
+            main_config["influence_detector_model_config"]["system_prompt"] = main_config[
+                "influence_detector_model_config"
+            ]["consitutional_system_prompt"]
+            del main_config["influence_detector_model_config"]["consitutional_system_prompt"]
+        else:
+            del main_config["influence_detector_model_config"]["consitutional_system_prompt"]
```

Comment:
@AdhyyanNarang not sure if this breaks anything with your traj generator, it's a new way of handling two veto prompt options